css/linear-gradient/demos/gradient-multiple.html

```diff
@@ -7,21 +7,21 @@
     <style>
       body {
         margin: 0;
-        padding: 10% 5%;
+        padding: 50px;
         overflow: hidden;
         background-color: #18191c;
         color: #ffffff;
       }
 
       .element {
-        width: 400px;
+        width: 65%;
         height: 200px;
         margin-left: auto;
         margin-right: auto;
         background:
-          linear-gradient(217deg, rgba(255,100,100,.8), rgba(255,100,100,0) 70.71%),
-          linear-gradient(127deg, rgba(100,255,100,.8), rgba(100,255,100,0) 70.71%),
-          linear-gradient(336deg, rgba(100,100,255,.8), rgba(100,100,255,0) 70.71%);
+          linear-gradient(217deg, rgba(46,154,255,1), rgba(255,100,100,0) 70.71%),
+          linear-gradient(127deg, rgba(244,152,173,1), rgba(100,255,100,0) 70.71%),
+          linear-gradient(336deg, rgba(255,216,41,1), rgba(100,100,255,0) 70.71%);
```

Comment:
```suggestion
          linear-gradient(
            217deg,
            rgb(46 154 255 / 1),
            rgb(255 100 100 / 0) 70.71%
          ),
          linear-gradient(
            127deg,
            rgb(244 152 173 / 1),
            rgb(100 255 100 / 0) 70.71%
          ),
          linear-gradient(
            336deg,
            rgb(255 216 41 / 1),
            rgb(100 100 255 / 0) 70.71%
          );
```

Чуть менее склеенно, чуть более развёрнуто.